src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -123,13 +123,6 @@ export function InternalDashboardTopNav({
     dashboardTitleRef.current?.focus();
   }, [title, viewMode]);
 
-  /**
```

Comment:
Why is this removed?

Reply:
If keep this, the top nav will disappear and reappear like below

https://github.com/user-attachments/assets/943f7f09-4fb2-4221-9ca8-ffe80c14d63c

And for `embed` mode, this function will ensure it works fine

https://github.com/elastic/kibana/blob/3bd11cead5b31312d019a05e61824548a680b5bf/packages/core/chrome/core-chrome-browser-internal/src/chrome_service.tsx#L111-L134

If there is a better way, I will fix it.